### DISCUSSION TITLE
What to name elements of programs array in site config files

### DISCUSSION THREAD

**Discussion by Greatnloopy** [iptv-org/community#1020] [Q&A] [answered]
Hello, I'm interested in trying to add some program information from tvpassport.com to the xmltv file but I don't know what to call the array elements so that they are formatted correctly when handled by the programs parser.

This is the current programs array for tvpassport.com looks like:

programs.push({
        title,
        sub_title,
        description: parseDescription($item),
        icon: parseIcon($item),
        category: parseCategory($item),
        rating: parseRating($item),
        actors: parseActors($item),
        guest: parseGuest($item),
        director: parseDirector($item),
        start,
        stop
      })

But I would like to add elements for new episode, live episode and episode number (possible others). Would something like this work?

newep: parseNewEp($item),
liveep: parseLiveEp($item),
epnum: parseEpNum($item)

Of course, the called functions would be written as well.  Thank you.

**Comment by PopeyeTheSai10r** [comment]
Looks like you are on the right path. 
You will need to update the `config.js` instead of the xml file.
The best way to know is by cloning the repo, making the changes locally and testing.

**Comment by freearhey** [comment, marked as answer]
You can find the full list of supported properties here: https://github.com/freearhey/epg-grabber?tab=readme-ov-file#program-object
